tests/hmm/pars.py

```diff
@@ -551,7 +559,7 @@
 
         "do_rochester_corrections": True, 
         "do_lepton_sf": True,
-        "do_geofit": True,
+        "do_geofit": False,
```

Comment:
Should this be turned back on?

Reply:
Hi Irene! Yes, I will turn it back on. sorry I turned it off for a moment to speed up the debugging.